schemas/dublin-core/pom.xml

```diff
@@ -42,6 +45,44 @@
           </execution>
         </executions>
       </plugin>
+
+      <!-- package up plugin folder as a zip -->
+      <plugin>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>plugin-assembly</id>
+            <phase>package</phase>
+            <goals><goal>single</goal></goals>
+            <inherited>false</inherited>
+            <configuration>
+             <appendAssemblyId>false</appendAssemblyId>
+             <descriptors>
+              <descriptor>src/assembly/schema-plugin.xml</descriptor>
+             </descriptors>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
```

Comment:
This looks quite similar to the one above no? Can we get rid of it?

Reply:
Yes, not sure why it was duplicated.